board_widgets.go

```diff
@@ -304,17 +304,19 @@ type LogStreamDefinition struct {
 
 // ManageStatusDefinition represents the definition for a Manage Status widget
 type ManageStatusDefinition struct {
-	Type            *string `json:"type"`
-	Query           *string `json:"query"`
-	Sort            *string `json:"sort,omitempty"`
-	Count           *int    `json:"count,omitempty"`
-	Start           *int    `json:"start,omitempty"`
-	DisplayFormat   *string `json:"display_format,omitempty"`
-	ColorPreference *string `json:"color_preference,omitempty"`
-	HideZeroCounts  *bool   `json:"hide_zero_counts,omitempty"`
-	Title           *string `json:"title,omitempty"`
-	TitleSize       *string `json:"title_size,omitempty"`
-	TitleAlign      *string `json:"title_align,omitempty"`
+	Type              *string `json:"type"`
+	SummaryType       *string `json:"summary_type"`
```

Comment:
Is this a required property or not? If not (which I assume is the case), it should be marked with `omitempty` (which is also done is `screen_widgets.go` in this PR).

Reply:
Good catch! I pushed a commit to fix it.